doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -974,32 +974,42 @@ Generation of induction principles with ``Scheme``
    .. insertprodn scheme_kind sort_family
 
    .. prodn::
-      scheme_kind ::= Equality for @reference
-      | {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
+      scheme_kind ::= {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
       sort_family ::= Set
       | Prop
       | SProp
       | Type
 
-  A high-level tool for automatically generating
-  (possibly mutual) :term:`induction principles <induction principle>`
-  for given types and sorts.
-  Each :n:`@reference` is a different inductive type identifier belonging to
-  the same package of mutual inductive definitions.
-  The command generates the :n:`@ident`\s as mutually recursive
-  definitions. Each term :n:`@ident` proves a general principle of mutual
-  induction for objects in type :n:`@reference`.
+   Generates (possibly mutual) :term:`induction principles <induction principle>`
+   for given types and sorts. Each :n:`@reference` is a different inductive type
+   identifier belonging to the same package of mutual inductive definitions. The
+   command generates the :n:`@ident`\s as mutually recursive definitions. Each
+   term :n:`@ident` proves a general principle of mutual induction for objects
+   in type :n:`@reference`.
 
-  :n:`@ident`
-    The name of the scheme. If not provided, the scheme name will be determined automatically
-    from the sorts involved.
+   :n:`@ident`
+      The name of the scheme. If not provided, the scheme name will be determined
+      automatically from the sorts involved.
```

Comment:
```suggestion
      The name of the scheme. If not provided, the scheme name is chosen
      based on :n:@sort_family`.
```

Reply:
The name is actually determined from the scheme kind and the sort family. I will detail that.

Reply:
An in-line example might be nice.

Reply:
I have mentioned that it depends on the `scheme_type` and `scheme_sort`. I have also detailed in a comment below why going into detail might not be worth it.